plugins/woocommerce/src/Internal/DataStores/Orders/OrdersTableDataStore.php

```diff
@@ -2918,16 +2919,31 @@ public function update_meta( &$object, $meta ) {
 	 *
 	 * @param WC_Abstract_Order $order Order object.
 	 * @param \WC_Meta_Data     $meta  Metadata object.
+	 *
+	 * @return bool True if changes were applied, false otherwise.
 	 */
 	protected function after_meta_change( &$order, $meta ) {
```

Comment:
The function name is a bit odd: It would be nicer to actually know what happens with the order/data store here rather than know when it is happening. But that's outside of this PR's changeset.